web/src/App.tsx

```diff
@@ -1,14 +1,106 @@
-import { Container, MantineProvider } from "@mantine/core";
+import { Container, MantineProvider, Button, Flex } from "@mantine/core";
 import "@mantine/core/styles.css";
 import React from "react";
 import { theme } from "./theme";
+import * as ReactDOM from "react-dom/client";
+import {
+  createBrowserRouter,
+  RouterProvider,
+  Outlet
+} from "react-router-dom";
+import img from "../public/zenith-logo.svg";
 
+interface NumberButtonProps {
+  digit: number,
+}
```

Comment:
let's use type instead.
types in TS are considered to be more reliable than interfaces.